test/ts/integration/examples/erc721_collateralized_simple_interest_terms_contract/scenarios/index.ts

```diff
@@ -0,0 +1,241 @@
+    interestRateFixedPoint: Units.interestRateFixedPoint(2.5),
+    amortizationUnitType: new BigNumber(1),
+    termLengthUnits: new BigNumber(4),
+    // Parameters for collateralization.
+    collateralToken: "REP",
+    collateralId: new BigNumber(0),
+    // Misc parameters.
+    debtorFee: Units.ether(0.001),
+    invokedByDebtKernel: true,
+    permissionToCollateralize: true,
+    principalTokenInRegistry: true,
+    collateralTokenInRegistry: true,
+    succeeds: true,
+    isCryptoKitty: false,
+    reverts: false,
+    termsContractParameters: (terms: SimpleInterestContractTerms) =>
+        SimpleInterestParameters.pack(terms),
+};
+
+export const DEFAULT_RETURN_COLLATERAL_ARGS = {
+    principalAmount: Units.ether(1),
+    interestRateFixedPoint: Units.interestRateFixedPoint(2.5),
+    amortizationUnitType: new BigNumber(1),
+    termLengthUnits: new BigNumber(4),
+    repaymentAmount: Units.ether(1.29),
+    debtorFee: Units.ether(0.001),
+    // Parameters for collateralization.
+    collateralToken: "MET",
+    collateralId: new BigNumber(0),
+    repaymentToken: (principalToken: DummyTokenContract, otherToken: DummyTokenContract) =>
+        principalToken,
+    debtOrder: (debtOrder: SignedDebtOrder) => debtOrder,
+    principalTokenInRegistry: true,
+    collateralTokenInRegistry: true,
+    isCryptoKitty: false,
+    succeeds: true,
+    reverts: false,
+};
+
+export const DEFAULT_SEIZE_COLLATERAL_ARGS = {
+    principalAmount: Units.ether(1),
+    interestRateFixedPoint: Units.interestRateFixedPoint(2.5),
+    amortizationUnitType: new BigNumber(1),
+    termLengthUnits: new BigNumber(4),
+    repaymentAmount: Units.ether(1.29),
+    debtorFee: Units.ether(0.001),
+    // Parameters for collateralization.
+    collateralToken: "MET",
+    collateralId: new BigNumber(0),
+    repaymentToken: (principalToken: DummyTokenContract, otherToken: DummyTokenContract) =>
+        principalToken,
+    debtOrder: (debtOrder: SignedDebtOrder) => debtOrder,
+    principalTokenInRegistry: true,
+    collateralTokenInRegistry: true,
+    isCryptoKitty: false,
+    secondsSinceFill: 0,
+    succeeds: true,
+    reverts: false,
+};
+
+export const DEFAULT_REGISTER_REPAYMENT_ARGS = {
+    principalAmount: Units.ether(1),
+    interestRateFixedPoint: Units.interestRateFixedPoint(2.5),
+    amortizationUnitType: new BigNumber(1),
+    termLengthUnits: new BigNumber(4),
+    repaymentAmount: Units.ether(1.29),
+    debtorFee: Units.ether(0.001),
+    // Parameters for collateralization.
+    collateralToken: "MET",
+    collateralId: new BigNumber(0),
+    repaymentToken: (principalToken: DummyTokenContract, otherToken: DummyTokenContract) =>
+        principalToken,
+    debtOrder: (debtOrder: SignedDebtOrder) => debtOrder,
+    repayFromRouter: true,
+    principalTokenInRegistry: true,
+    collateralTokenInRegistry: true,
+    isCryptoKitty: false,
+    succeeds: true,
+    reverts: false,
+};
+
+export interface RegisterRepaymentScenario {
```

Comment:
can you not create a base scenario that defines the base params, and then have these other scenarios extend the base scenario. would reduce a lot of duplicate text here.